From 4d8e7c049a2e97476fbff323c9a8ba39c0f4b96a Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Wed, 8 Jan 2014 04:41:02 -0600
Subject: [PATCH 031/466] fsl_qbman: Fix the checkpatch error type
 TRAILING_WHITESPACE

ERROR:TRAILING_WHITESPACE: trailing whitespace

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Change-Id: I4d2a46862edc2066a9aabf236cb22e71bfa0e375
Reviewed-on: http://git.am.freescale.net:8181/7769
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Reviewed-by: Emilian Medve <Emilian.Medve@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/staging/fsl_qbman/fsl_usdpaa.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_qbman/fsl_usdpaa.c b/drivers/staging/fsl_qbman/fsl_usdpaa.c
index 7e3b34a..44c5464 100644
--- a/drivers/staging/fsl_qbman/fsl_usdpaa.c
+++ b/drivers/staging/fsl_qbman/fsl_usdpaa.c
@@ -331,7 +331,7 @@ static int init_qm_portal(struct qm_portal_config *config,
 	}
 
 	/* Initialize the EQCR */
-	if (qm_eqcr_init(portal, qm_eqcr_pvb, 
+	if (qm_eqcr_init(portal, qm_eqcr_pvb,
 			qm_eqcr_get_ci_stashing(portal), 1)) {
 		pr_err("Qman EQCR initialisation failed\n");
 		return 1;
@@ -1499,7 +1499,7 @@ __init void fsl_usdpaa_init_early(void)
 		phys_size = 0;
 		return;
 	}
-	phys_start = __memblock_alloc_base(phys_size, 
+	phys_start = __memblock_alloc_base(phys_size,
 					   largest_page_size(phys_size),
 					   MEMBLOCK_ALLOC_ACCESSIBLE );
 	if (!phys_start) {
-- 
1.7.10.4

